cmd/ci-operator-prowgen/main_test.go

```diff
@@ -609,7 +603,7 @@ func TestFromCIOperatorConfigToProwYaml(t *testing.T) {
           valueFrom:
             configMapKeyRef:
               key: branch.yaml
```

Comment:
This is wrong, yeah?

Reply:
You mean wrong by not having `$org-$repo-$branch.yaml` format? Yeah. 

It will still extract the branch name right - it will attempt to remove a `super-duper-` prefix and `branch` will remain :)

Reply:
Fun side-effect! Backwards compatiblity